pulp_rpm/tests/functional/constants.py

```diff
@@ -53,11 +50,21 @@
     'summary': 'A dummy package of bear',
     'rpm_license': 'GPLv2',
     'rpm_group': 'Internet/Applications',
-    'rpm_vendor': "",
+    'rpm_vendor': '',
     # TODO: Complete this information once we figure out how to serialize everything nicely
 }
 """The metadata for one RPM package."""
 
+RPM_UPDATEINFO_URL = urljoin(
+    PULP_FIXTURES_BASE_URL,
+    'rpm-references-updateinfo/'
+)
```

Comment:
I don't think it should be named this.  It might lead to confusion if someone thinks that the ```RPM_UPDATED_UPDATEINFO_FIXTURE_URL``` fixture is updated compared to this one, when actually it's "updated" vs the standard signed/unsigned fixture.  This one is actually completely different, it includes reference information that neither the signed/unsigned fixtures nor the "rpm-updated-updateinfo" fixture have.

Also ```rpm-pkglists-updateinfo```